test/test_nn.py

```diff
@@ -2108,12 +2108,13 @@ def test_threshold_int(self):
         expected = torch.tensor([99, 99, 99, 99, 1, 2, 3])
         self.assertEqual(F.threshold(x, 0, 99), expected)
 
-    def test_threshold_bfloat16(self):
+    def test_threshold_bfloat16_half(self):
```

Comment:
Can we use `@dtype` decorator?

Reply:
Can't use @dtype here because it is in the scope of TestNN not TestNNDeviceType.